src/containers/edit-profile/profile-tab/profile-tab-form/ProfileTabForm.tsx

```diff
@@ -116,10 +93,7 @@ const ProfileGeneralTab: FC<ProfileTabProps> = ({ user }) => {
   }
 
   const handleRemovePhoto = () => {
-    handleProfileData({
-      ...updatedData,
-      photo: null
-    })
+    setPhoto(null)
```

Comment:
When user without photo clicks remove photo button, nothing changes for user, but the pop-up still appears after click on one of navigation buttons. The same when user without photo sets photo and without saving clicks remove photo button. Is that a desired behaviour? Maybe would be good to set photo to `null` only if the user has something inside of `user.photo`, and set it to initial photo state (`{ src: '', name: '' }` in `ProfileTab`) in other case? Something like this: `user.photo ? null : { src: '', name: '' }`.

Reply:
It's not a desired behavior indeed, I missed this case. Changed it based on your suggestion, thanks a lot.